src/QMCDrivers/QMCDriverNew.cpp

```diff
@@ -545,11 +545,10 @@ void QMCDriverNew::endBlock()
     total_accept_ratio += crowd->get_accept_ratio() * crowd->get_estimator_manager_crowd().get_block_weight();
     cpu_block_time += crowd->get_estimator_manager_crowd().get_cpu_block_time();
   }
-  total_accept_ratio /= total_block_weight;
```

Comment:
It seems the old code is wrong in many regards.
1. blockweight != the number of walkers in a block.
2. energies and observables should be averaged by weight but acceptance ratio should be averaged by walker counts.
3. Block CPU time, should we even average? or just collect the rank 0 time?

Reply:
Or, for Block CPU time, take the max value over the crowds?

Reply:
max value is desired but it may add extra synchronization.

Reply:
A hazard of poorly defined terms!

* Acceptance ratio is computed over #walkers with no weight contribution.
* Blockweight is meant to be the total weight of walkers in the block. For a healthy load balanced run it will be similar to the total number of walkers.
* Block CPU is just a timer. I suggest we not worry about computing the max value for now and just report the average to avoid extra syncs. For a healthy run this is good enough.

Reply:
The accept ratio is handled properly in #2672.  Statistics when walkers are transferred are still somewhat off.
BlockCPU is currently the total, could be average easily,

Reply:
@PDoakORNL Thanks. If we merge this will the conflicts tolerable or would you prefer we not merge this?

Reply:
Please don't merge this. It conflicts and the ratio normalization is wrong its better not in the history.